coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -0,0 +1,36 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+
+class FirebaseClientWrapper:
+    def __init__(self, client):
+        """Inits FirebaseClientWrapper
+
+        Args:
+            Firestore client (google.cloud.firestore.Firestore): Client for interacting with Google Cloud Firestore.
+        """
+        self._client = client
+
+    @classmethod
+    def init_client(cls, crypto_token_path, app_name):
```

Comment:
Please provide a default app name e.g.
```suggestion
    def init_client(cls, crypto_token_path, app_name="CodaV2Client"):
```
This is so initialisation is easy for the common case of only needing to connect to one Coda firestore